src/BloomBrowserUI/bookEdit/js/readerTools.js

```diff
@@ -265,30 +268,38 @@ function loadExternalLink(url) {
  * We need to check the size of the decodable reader tool pane periodically so we can adjust the height of the word list
  * @global {number} previousHeight
  */
-function resizeWordList() {
+function resizeWordList(startTimeout) {
+    if (startTimeout === void 0) { startTimeout = true; }
```

Comment:
I assume this is giving startTimeout a default value of true, allowing the method to be called with or without an argument.